src/internal.cc

```diff
@@ -86,8 +86,7 @@ internalStaticCheck(const SBuf &urlPath)
 bool
 ForSomeCacheManager(const SBuf &urlPath)
 {
-    static const SBuf mgrPfx("/squid-internal-mgr");
```

Comment:
This was the poorly duplicated prefix definition that resulted in `Squid BUG: assurance failed: tok.skip(internalMagicPrefix)` ERRORs.